projects/canopy/src/lib/link-menu/link-menu.stories.ts

```diff
@@ -0,0 +1,76 @@
+import { Meta, Story } from '@storybook/angular';
+import { moduleMetadata } from '@storybook/angular';
+
+import { LgLinkMenuModule } from './link-menu.module';
+import { LgLinkMenuComponent } from './link-menu.component';
+import { notes } from './link-menu.notes';
+
+const template = `
+<lg-link-menu>
+  <a href="" *ngFor="let item of menuItems">
+    <lg-link-menu-item>
+      <lg-link-menu-item-heading>{{ item.title }}</lg-link-menu-item-heading>
+      <lg-link-menu-item-content *ngIf="item.description">
+        {{ item.description }}
+      </lg-link-menu-item-content>
+    </lg-link-menu-item>
+  </a>
+</lg-link-menu>`;
+
+// This default export determines where your story goes in the story list
+export default {
+  title: 'Components/Link Menu',
+  component: LgLinkMenuComponent,
+  decorators: [
+    moduleMetadata({
+      imports: [ LgLinkMenuModule ],
+    }),
+  ],
+  parameters: {
+    docs: {
+      description: {
+        component: notes,
+      },
+    },
+  },
+  argTypes: {
+    class: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+} as Meta;
+
+const linkMenuTemplate: Story<LgLinkMenuComponent> = (args: LgLinkMenuComponent) => ({
+  props: args,
+  template,
+});
+
+function getDefaultMenuItems() {
```

Comment:
Could you please add a return type to this function please? Also it could be marked as private

Reply:
> @aaronBery there seem to be a bit of jumping around caused by the thick underline that is added on hover. Maybe the underline could always be there but set as transparent?
> 
>  Screen.Recording.2022-05-24.at.16.12.21.mov

Hmmm i applied a negative margin which i thought solved that but I hadn't tested with a second list item and you're right it still exists. in terms of the border there's already a grey border we need in a non hover state so i might need to think of another solution...

Reply:
> Could you please add a return type to this function please? Also it could be marked as private

I can add a return type sure. Not sure about marking a function private though...